src/base_classes/mo_space_info.h

```diff
@@ -157,13 +160,21 @@ class MOSpaceInfo {
         {"GENERALIZED PARTICLE", {"ACTIVE", "RESTRICTED_UOCC"}},
         {"CORE", {"RESTRICTED_DOCC"}},
         {"VIRTUAL", {"RESTRICTED_UOCC"}}};
+
     /// The names of the orbital spaces
     std::vector<std::string> space_names_;
+
     /// The map from all MO to the correlated MOs (excludes frozen core/virtual)
     std::vector<size_t> mo_to_cmo_;
+
+    // ==> Class functions <==
+    /// Read information about each elementary space from the psi Options object
+    std::pair<SpaceInfo, bool> read_mo_space(const std::string& space, psi::Options& options);
 };
 
-} // namespace forte
+std::shared_ptr<MOSpaceInfo> make_mo_space_info(std::shared_ptr<psi::Wavefunction> ref_wfn,
+                                                psi::Options& options);
```

Comment:
Can here accept ForteOption arg instead of psi::Options?

Reply:
Should be easy, I'll push it right away.